src/repository/storage/postgres/orm/sequelize/note.ts

```diff
@@ -244,6 +245,21 @@ export default class NoteSequelizeStorage {
     return note;
   };
 
+  /**
+   * Gets note list by creator id
+   *
+   * @param creatorId - note creator id
+   * @returns { Promise<NoteList | null> } note
+   */
+  public async getNoteListByCreatorId(creatorId: NoteCreatorId): Promise<NoteList | null> {
```

Comment:
I think, that in can not return null, it will be `[]`.

Reply:
Made a condition for returning null if the array is empty